thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -4334,7 +4335,11 @@ IRowStream *mergeLocalAggs(Owned<IHashDistributor> &distributor, CActivityBase &
         {
             for (;;)
             {
-                OwnedConstThorRow row = input->nextRow();
+                OwnedConstThorRow row;
+                {
+                    BlockedActivityTimer t(activity.totalCycles, activity.queryTimeActivities());
```

Comment:
not directly related to new feature, but we did discuss changing 'totalCycles' to something else, as it's more than just that, I think called that because it used to be just a cycle_t.
Should be called something like slaveTimerStats or similar.